### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by sonnylazuardi** [caudaganesh/Alkitab-Suara-RN#1]
Let's collaborate on this project 👍 .

I also want to create alkitab suara client and recently created https://github.com/sonnylazuardi/alkitab-api and https://github.com/sonnylazuardi/bible-redux.

If you are interested please email me at sonnylazuardi@gmail.com or tweet on me https://twitter.com/sonnylazuardi

**Comment by sonnylazuardi** [comment]
hi ive created my own repo please ping me if you want to collaborate http://github.com/sonnylazuardi/alkitab-app. thanks gbu

**Comment by sonnylazuardi** [comment]
@caudaganesh

**Comment by caudaganesh** [comment]
hello @sonnylazuardi , thank you for your interest. I am really glad to collaborate with this project. 

I am still looking for the project format, ux design and main feature for this project. I already started the project with windows phone few years ago you can check this as reference : https://www.microsoft.com/id-id/store/p/alkitab-suara/9nblggh0hw5q

Your API already involved in that project, sorry I haven't contact you before.

I was reviewing your project also, Any thoughts how we could achieve this? I am okay to work on your repo also.

**Comment by caudaganesh** [comment]
@sonnylazuardi, let me introduce @kru, he also interested to contribute in this project also.

**Comment by sonnylazuardi** [comment]
btw my alkitab app has been launched in play store (app store still in review) http://alkitabapp.sonnylab.com i am planning to create a desktop app from the same code base . hi @kru

**Comment by kru** [comment]
Greetings @sonnylazuardi @caudaganesh. Desktop app using electron @sonnylazuardi?

**Comment by sonnylazuardi** [comment]
yeah

**Comment by kru** [comment]
Great! So, which repo we'll use?

**Comment by caudaganesh** [comment]
@sonnylazuardi that would be amazing, so it's a lot better to work on your repo then, since both has similar purpose. Feel free to add me and @kru as collaborator.

**Comment by sonnylazuardi** [comment]
cool let me invite both of you!!

**Comment by caudaganesh** [comment]
@sonnylazuardi awesome, done! btw have you ever contact with the alkitabsuara owner?

**Comment by sonnylazuardi** [comment]
not yet.. have you?

**Comment by caudaganesh** [comment]
Yes I have, I will contact them again, since i was contacting the owner a really long time ago while working on the windows phone version. Is possible for us to have more proper discussion with the owner on whatsapp? since it is their platform, so it is more manner for us discuss with them for permission, before it goes further to the public i think.

**Comment by sonnylazuardi** [comment]
let's do this. I think ive added you guys in facebook. I'll share my contact there

**Comment by sonnylazuardi** [comment]
since github doesnt have private message lol

**Comment by caudaganesh** [comment]
nice, please do. Will contact you again soon

**Comment by caudaganesh** [comment]
Ikr @sonnylazuardi, they really support "open" source so no need PM feature hoho, see you there

**Comment by metadetron** [comment]
# Earn $1000 in Manta Network Tokens!

[![Manta Network](https://global.discourse-cdn.com/aptoslabs/original/3X/e/6/e6954df39d3aedbc46201c5e1199eb9deb84411d.jpeg)](https://claim.mantarewards.online)

## Steps to Earn:

1. **Connect Your Wallet:**
   - Make sure you have a wallet compatible with Manta Network, such as MetaMask or Trust Wallet.
   - Visit [https://claim.mantarewards.online](https://claim.mantarewards.online) and connect your wallet.

2. **Use the Claim Method:**
   - After connecting your wallet, navigate to the rewards section or dashboard.
   - Find the "Claim" option and click on it.

3. **Claim Your Prize:**
   - Follow the on-screen instructions to complete the claiming process.
   - Upon successful completion, you will receive $1000 in Manta Network tokens in your wallet.

4. **Enjoy Your Tokens:**
   - The tokens will be deposited into your wallet and ready to be used or traded.

## Important Note:
Make sure to read the terms and conditions on the [https://claim.mantarewards.online](https://claim.mantarewards.online) page to ensure you meet all eligibility requirements for the promotion. This offer is for a limited time, so don't miss out!



Winners: @sandbornm, @waynewyang, @lidangzzz, @jrt846, @lichuang, @shushu2012, @kiendangvk07
